Template/Pages/About Us.html

```diff
@@ -1,13 +1,11 @@
 <!--
  Title: About Us
  -->
-<p>Staff:</p>
+<h4>Staff:</h4>
 <p>Pastor - Rev. Steven Menshenfriend</p>
 <p>Office Administrator - Joyce Hancock</p>
-<p>OUR VISION STATEMENT</p>
-<p>"Growing and Reaching Out Together"</p>
-<p>OUR MISSION STATEMENT:</p>
-<p>"To Know Christ and To Make Him Known"</p>
+<h4>OUR VISION STATEMENT: </h4><p>"Growing and Reaching Out Together"</p>
```

Comment:
Do not confuse capitals with upper case. Capitals are letters which are grammatically required to be that way, and thus they carry semantic meaning. Examples are proper names, acronyms, or the beginning of sentences. Upper case is the letter forms of capital letters.

Some fonts only use upper case letters, *but this is a matter of the font*. During things like copying, pasting and searching, you do not want it to be considered, just like you don’t want bold or italics to be treated as different letters by the search operation.

**Always write the raw text with grammatically proper casing.** Then, if you want it to use an upper case font style, apply CSS:

```css
text-transform: uppercase;
```

***

The colons shouldn’t be in the headings here either.